coalib/settings/ConfigurationGathering.py

```diff
@@ -122,6 +124,16 @@ def warn_config_absent(sections, argument, log_printer):
                          "to give the `--{}` argument?".format(argument))
 
 
+def configure_logging_from_json(filename):
+    """
+    Configures logging module based on parameters from JSON file.
+
+    :param filename:    Name of a file with logging configuration.
+    """
+    with open(filename) as cnf_file:
+        logging.config.dictConfig(json.load(cnf_file))
```

Comment:
What's the benefit of having a separate JSON file instead of just keeping the config somewhere as a Python dictionary?